test/elixir/test/config_test.exs

```diff
@@ -174,4 +174,11 @@ defmodule ConfigTest do
       set_config(context, section, "wohali", "rules", 403)
     end)
   end
+
+  test "Reload config", context do
+    url = "#{context[:config_url]}/_reload"
+    resp = Couch.post(url)
+
+    assert resp.status_code == 200
```

Comment:
It might be nice to actually demo changing a config value in e.g. a .ini file and see it change with the reload, but I'm not sure if there's a simple way to make that filesystem modification via elixir test.

Reply:
yes - I couldn't see a good way to test this, though that test should perhaps exist for `couch_config` rather than this endpoint specifically.

Reply:
the admin case is interesting - I wonder if that's to do with couchdb_config not triggering `config_event` notifications on reload.